src/styles/block/stopwatch.scss

```diff
@@ -0,0 +1,66 @@
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 80vmin;
+  height: 80vmin;
+  line-height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  margin: 69px auto;
+
+  &__minute-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 48%;
+    transform-origin: bottom center;
+    transform: rotate(0deg);
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+
+    animation: rotate-minute 600s steps(60) infinite; /* Минутная стрелка за час */
+  }
+
+  &__second-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 49%;
+    transform-origin: bottom center;
+    transform: rotate(0deg);
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+
+    animation: rotate-second 9997ms linear infinite; /* Секундная стрелка за минуту */
+  }
+
+  &__center-point {
+    position: absolute;
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+}
+
+
+
+
+
+
+
+.stopwatch.stopwatch--speed-up {
+  .stopwatch__second-hand {
+    animation: rotate-second 9997ms linear infinite;
+  }
+
+  .stopwatch__minute-hand {
+    animation: rotate-minute 600s steps(60) infinite;
+  }
```

Comment:
The `stopwatch--speed-up` modifier seems to be redefining the same animation properties with the same values for `.stopwatch__second-hand` and `.stopwatch__minute-hand` as already defined in the base `.stopwatch` block. This is redundant and can be removed unless there's a specific reason for this modifier to exist.